CONTRIBUTING.md

```diff
@@ -139,7 +139,16 @@ A guide on commonly used labels added to issues and pull requests:
 
 ## Testing Your Code
 
-- You can test your code locally along with your changes using: `yarn storybook` or `npm run storybook`. This will open the storybook in your browser .
+- You can test your code locally along with your changes using: `yarn storybook` or `npm run storybook`. This will open the storybook in your browser.
+
+This project contains tests that are executed using Jest. The bulk of the component tests are written with React Testing Library. These tests can be run using `yarn test` or `npm run test`. When changes to functionality are made, tests should always be included. The following best practices should be observed when writing Jest tests with React Testing Library:
```

Comment:
```suggestion
This project contains unit tests executed by Jest. The bulk of the component tests are written with [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/) to simulate end user behavior and focus on testing functionality instead of implementation.

Jest tests are run using `yarn test` or `npm run test`. When changes to functionality are made, tests should always be included covering the functionality introduced.

The following best practices should be observed when writing Jest tests with React Testing Library:
```

Reply:
```suggestion
**Automated Tests**
This project contains tests that are executed using Jest. The bulk of the component tests are written with React Testing Library. These tests can be run using `yarn test` or `npm run test`. When changes to functionality are made, tests should always be included. The following best practices should be observed when writing Jest tests with React Testing Library:
```